apps/mobile/src/App.tsx

```diff
@@ -95,7 +58,9 @@ function App(): JSX.Element {
 export default function _(): JSX.Element {
   return (
     <ThemeProvider>
-      <App />
+      <AnimatedSplashScreen>
+        <App />
```

Comment:
Will this really prevent rendering the whole app until splashScreen is finished?